src/main/java/core/basesyntax/service/operations/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        //storage = (Storage) Storage.fruits;

```

Comment:
The commented out line of code should be removed for cleaner code.

Reply:
ok, done